packages/pigment-css-react/src/utils/extendTheme.ts

```diff
@@ -167,5 +177,13 @@ export function extendTheme<
     };
   };
 
+  finalTheme.unstable_sxConfig = defaultSxConfig;
+  finalTheme.unstable_sx = function sx(props) {
```

Comment:
I tried but I think doing it all together with Material UI, System in v6 might be better, otherwise some places have to support both `theme.sx` and `theme.unstable_sx`.

```diff
diff --git a/packages/mui-system/src/styleFunctionSx/styleFunctionSx.js b/packages/mui-system/src/styleFunctionSx/styleFunctionSx.js
index 151f117c41..fd033f8a6e 100644
--- a/packages/mui-system/src/styleFunctionSx/styleFunctionSx.js
+++ b/packages/mui-system/src/styleFunctionSx/styleFunctionSx.js
@@ -81,7 +81,7 @@ export function unstable_createStyleFunctionSx() {
       return null; // Emotion & styled-components will neglect null
     }
 
-    const config = theme.unstable_sxConfig ?? defaultSxConfig;
+    const config = theme.sxConfig || theme.unstable_sxConfig || defaultSxConfig;
 
     /*
      * Receive `sxInput` as object or callback
diff --git a/packages/pigment-css-react/src/utils/extendTheme.ts b/packages/pigment-css-react/src/utils/extendTheme.ts
index cd1df7cb46..b36e45a50c 100644
--- a/packages/pigment-css-react/src/utils/extendTheme.ts
+++ b/packages/pigment-css-react/src/utils/extendTheme.ts
@@ -81,8 +81,8 @@ export type ExtendTheme<
       css: Record<string, string | number>;
       selector: string | Record<string, any>;
     };
-    unstable_sxConfig?: SxConfig;
-    unstable_sx?: (
+    sxConfig?: SxConfig;
+    sx?: (
       props: SxProps<
         Options['tokens'] & {
           vars: Options['tokens'];
@@ -177,8 +177,8 @@ export function extendTheme<
     };
   };
 
-  finalTheme.unstable_sxConfig = defaultSxConfig;
-  finalTheme.unstable_sx = function sx(props) {
+  finalTheme.sxConfig = defaultSxConfig;
+  finalTheme.sx = function sx(props) {
     return styleFunctionSx({
       sx: props,
       theme: this,
diff --git a/packages/pigment-css-react/src/utils/processCssObject.ts b/packages/pigment-css-react/src/utils/processCssObject.ts
index 190feac4d6..adc4911931 100644
--- a/packages/pigment-css-react/src/utils/processCssObject.ts
+++ b/packages/pigment-css-react/src/utils/processCssObject.ts
@@ -6,7 +6,9 @@ export function processCssObject(
   themeArgs?: PluginCustomOptions['themeArgs'],
   skipSx = true,
 ) {
-  const processedObj = skipSx ? cssObj : themeArgs?.theme?.unstable_sx?.(cssObj) || cssObj;
+  const processedObj = skipSx
+    ? cssObj
+    : themeArgs?.theme?.sx?.(cssObj) || themeArgs?.theme?.unstable_sx?.(cssObj) || cssObj;
   const className = css(processedObj as any);
   return cache.registered[className];
 }

```

Reply:
Or maybe this PR can be a part of stabilizing `unstable_sx` to `sx`.